ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/helpers/WindowHelper.kt

```diff
@@ -1,18 +1,69 @@
 package com.revenuecat.purchases.ui.revenuecatui.helpers
 
+import androidx.annotation.VisibleForTesting
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.ReadOnlyComposable
+import androidx.compose.ui.platform.LocalConfiguration
+import androidx.compose.ui.platform.LocalContext
+import androidx.window.core.layout.WindowHeightSizeClass
 import androidx.window.core.layout.WindowSizeClass
 import androidx.window.core.layout.WindowWidthSizeClass
 import androidx.window.layout.WindowMetricsCalculator
+import com.revenuecat.purchases.ui.revenuecatui.PaywallMode
+import com.revenuecat.purchases.ui.revenuecatui.data.PaywallState
+import com.revenuecat.purchases.ui.revenuecatui.isFullScreen
 
 @Composable
 @ReadOnlyComposable
-internal fun computeWindowWidthSizeClass(): WindowWidthSizeClass? {
-    val activity = LocalActivity.current ?: return null
-    val metrics = WindowMetricsCalculator.getOrCreate().computeCurrentWindowMetrics(activity)
-    val width = metrics.bounds.width()
-    val height = metrics.bounds.height()
-    val density = activity.resources.displayMetrics.density
-    return WindowSizeClass.compute(width / density, height / density).windowWidthSizeClass
+internal fun computeWindowWidthSizeClass(): WindowWidthSizeClass {
+    return windowSizeClass().windowWidthSizeClass
+}
+
+@Composable
+@ReadOnlyComposable
+internal fun computeWindowHeightSizeClass(): WindowHeightSizeClass {
+    return windowSizeClass().windowHeightSizeClass
+}
+
+@Composable
+@ReadOnlyComposable
+internal fun PaywallState.Loaded.shouldUseLandscapeLayout(): Boolean {
+    return shouldUseLandscapeLayout(mode = templateConfiguration.mode)
+}
+
+@Composable
+@ReadOnlyComposable
+private fun shouldUseLandscapeLayout(mode: PaywallMode): Boolean {
+    return shouldUseLandscapeLayout(mode, computeWindowHeightSizeClass())
+}
+
+@VisibleForTesting
```

Comment:
This could be:
```suggestion
@VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
```
This way the IDE will warn if we try to use it.